libcudacxx/include/cuda/std/__atomic/wait/polling.h

```diff
@@ -24,7 +24,7 @@
 #include <cuda/std/__atomic/order.h>
 #include <cuda/std/__atomic/scopes.h>
 #include <cuda/std/__atomic/types.h>
-#include <cuda/std/detail/libcxx/include/__threading_support>
+#include <cuda/std/__atomic/wait/threading_support.h>
```

Comment:
Do not like this path. This header isn't atomics-specific, it should go into its own place.

Reply:
I moved it to `__thread`